src/picknik_ur_gazebo_config/launch/sim/hardware_sim.launch.py

```diff
@@ -59,7 +73,15 @@ def path_pattern_change_for_ignition(urdf_string):
 
 
 def generate_simulation_description(context, *args, **settings):
-    world_name = settings.get("gazebo_world_name", "space_station.sdf")
```

Comment:
If we don't want to switch to other world files as a launch argument (which probably isn't necessary at this point), we can get rid of these options.